### ISSUE TITLE
ACL add and delete for same set of emails on every run

### ISSUE THREAD

**Issue by rfdonnelly** [rfdonnelly/scma-gcal-sync#31]
On every run, I get the following message (emails have been anonymized) for the same two users.

```
2022-02-02T03:41:28.586925Z  INFO scma_gcal_sync::output::gcal: Determined sync operations ops.len=4 ops=[Insert("user-a-alias@example.com"), Insert("user-b@gmail.com"), Delete("user-a@example.com"), Delete("user-b@googlemail.com")]
```

I suspect that Google Calendar is resolving email aliases behind the scenes.  For example, if I insert `user-a-alias@example.com` Google Calendar returns a response that it inserted `user-a-alias@example.com`.  But when I do an `acl.list`, I don't get back `user-a-alias@example.com` .  Instead, I get back `user-a@example.com`.  In between the `acl.insert` response and and the `acl.list`, Google Calendar resolved the email alias and replaced the ACL rule with the resolved email address.

Until this is resolved, the calendar will be removed from these users' Google Calendar on every run and therefore unusable.

The first solution that comes to mind is an email map that maps an SCMA member's email address to the resolved email address. 
 The email map would be represented as a list of key-value pairs.  Or JSON.  Something like:

```json
{
  "user-a-alias@example.com": "user-a@example.com",
  "user-b@gmail.com": "user-b@googlemail.com"
}
```

**Comment by rfdonnelly** [comment]
Another possible solution is to use the following heuristic: if the number of additions is equal to the number of deletions then skip the deletions.

**Comment by rfdonnelly** [comment]
Fixed and deployed in [v2.0.0-rc.2](https://github.com/rfdonnelly/scma-gcal-sync/releases/tag/v2.0.0-rc.2)

**Comment by rfdonnelly** [comment]
Could not use the heuristic solution.  If the sync ops also contained a legitimate insertion or deletion it would have invalidated the heuristic and caused a deletion of valid users.
